build/ci/Makefile

```diff
@@ -7,6 +7,8 @@
 ROOT_DIR = $(CURDIR)/../..
 GO_MOUNT_PATH ?= /go/src/github.com/elastic/cloud-on-k8s
 
+IMAGE ?= docker.elastic.co/eck/eck-ci:$(shell md5sum $(ROOT_DIR)/operators/Gopkg.lock $(ROOT_DIR)/build/ci/Dockerfile $(ROOT_DIR)/local-volume/Gopkg.lock | awk '{print $$1}' | md5sum | awk '{print $$1}')
```

Comment:
Would it be possible to do hash calculation only here? We could only pass registry url and decide whether to build or not in each target. If we care about stage separation on jenkins side, we could call that decide&build target from jenkins every time.

Reply:
Fixed